plugins/module_utils/ome.py

```diff
@@ -101,6 +102,15 @@ def __init__(self, module_params=None, req_session=False):
         self.session_id = None
         self.protocol = 'https'
         self._headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+        try:
+            data = socket.getaddrinfo(self.hostname, self.port)
+            lastuple = data[-1]
+            self.hostname = lastuple[-1][0]
+            if "AF_INET6" == data[0][0]._name_:
+                self.hostname = "[{0}]".format(self.hostname)
+        except Exception:
+            msg = "Unable to resolve hostname or IP with OME {0}.".format(self.hostname)
```

Comment:
Better to provide a more generic message, instead of `OME` what if provided an `OMEM` IP address or hostname, still it returns with `OME`.

Reply:
Will change to this -> "Unable to resolve hostname or IP {0}."